[ 20-11-2019 :: 22:29:47 ] [  ERROR ] Unable To Create Connection With provisional_student_db
[ 20-11-2019 :: 23:11:09 ] [  INFO  ] provisional_student_db Connected Successfully
[ 20-11-2019 :: 23:11:12 ] [  INFO  ] <pymongo.results.InsertOneResult object at 0x7feb88023be0>
[ 20-11-2019 :: 23:11:12 ] [  ERROR ] Enrollment - 03620802717 Successfully Inserted at user_info Collection in provisional_student_db
[ 20-11-2019 :: 23:11:13 ] [  INFO  ] otp_db Connected Successfully
[ 20-11-2019 :: 23:11:15 ] [  INFO  ] For User ID - 03620802717 Duplicate Entry Found at generated_otp Collection in otp_db
[ 20-11-2019 :: 23:11:16 ] [  INFO  ] <pymongo.results.DeleteResult object at 0x7feb81779a00>
[ 20-11-2019 :: 23:11:16 ] [  INFO  ] User_ID - 03620802717 Removed Successfully from generated_otp Collection in otp_db
[ 20-11-2019 :: 23:11:16 ] [  INFO  ] <pymongo.results.InsertOneResult object at 0x7feb81779460>
[ 20-11-2019 :: 23:11:16 ] [  INFO  ] For User_ID - 03620802717 OTP Inserted Successfully at generated_otp Collection in otp_db
[ 20-11-2019 :: 23:11:16 ] [  INFO  ] For User ID - 03620802717 Duplicate Entry Found at generated_otp Collection in otp_db
[ 20-11-2019 :: 23:11:17 ] [  INFO  ] <pymongo.results.DeleteResult object at 0x7feb81768cd0>
[ 20-11-2019 :: 23:11:17 ] [  INFO  ] User_ID - 03620802717 Removed Successfully from generated_otp Collection in otp_db
[ 20-11-2019 :: 23:11:17 ] [  INFO  ] <pymongo.results.InsertOneResult object at 0x7feb81768550>
[ 20-11-2019 :: 23:11:17 ] [  INFO  ] For User_ID - 03620802717 OTP Inserted Successfully at generated_otp Collection in otp_db
